base/bitarray.jl

```diff
@@ -1375,9 +1375,81 @@ function (>>>)(B::BitVector, i::UInt)
     return A
 end
 
+"""
+    >>(B::BitVector, n) -> BitVector
+
+Right bit shift operator, `B >> n`. For `n >= 0`, the result is `B`
+with elements shifted `n` positions forward, filling with `false`
+values. If `n < 0`, elements are shifted backwards. Equivalent to
+`B << -n`.
```

Comment:
I wrote it this way because if `n` is positive or negative, `B >> n` is equivalent to `B << -n`. Hence the separate statement.